.eslintrc.json

```diff
@@ -18,16 +18,7 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off",
-    "react/no-unstable-nested-components": "off",
-    "react/no-namespace": "off",
-    "import/no-relative-packages": "off",
-    "import/no-import-module-exports": "off",
-    "react/no-unused-class-component-methods": "off",
-    "react/no-invalid-html-attribute": "off",
-    "react/no-arrow-function-lifecycle": "off",
-    "react/prefer-exact-props": "off",
-    "react/function-component-definition": "off"
+    "no-shadow": "off"
```

Comment:
- [ ] When I run the server I am getting this error, It is because you did not install linters dependencies please fix that you can install them:
` "devDependencies": {
    "@babel/core": "^7.15.5",
    "@babel/eslint-parser": "^7.15.7",
    "@babel/plugin-syntax-jsx": "^7.14.5",
    "@babel/preset-react": "^7.14.5",
    "eslint": "^7.32.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-plugin-import": "^2.24.2",
    "eslint-plugin-jsx-a11y": "^6.4.1",
    "eslint-plugin-react": "^7.26.1",
    "eslint-plugin-react-hooks": "^4.2.0",
    "react-test-renderer": "^17.0.2",
    "stylelint": "^13.13.1",
    "stylelint-config-standard": "^21.0.0",
    "stylelint-csstree-validator": "^1.9.0",
    "stylelint-scss": "^3.21.0"
  }`

![complie](https://user-images.githubusercontent.com/44745855/148434014-9f8309e8-d3be-44a2-9cd9-496b8ee32bc0.png)